docs/installation/setup-ide.md

```diff
@@ -70,15 +70,26 @@ The following Visual Studio configuration shows a typical configuration.
 {#setup-rider}
 ## Setup Rider
 
-% TODO
-```{admonition} Rider plugin is not available yet
-:class: error
-
-The Reqnroll Rider plugin has not yet been ported and released. Please, come back later or help contribute to it in our [open-source GitHub project](https://github.com/reqnroll/Reqnroll.Rider).
+In order to use Reqnroll with Rider, you need to install the [Reqnroll for Rider](https://plugins.jetbrains.com/plugin/24012-reqnroll-for-rider) extension.
 
-We recommend using the Visual Studio 2022 or the Visual Studio Code integration until the plugin will be ready. The SpecFlow Rider plugin can also be used with limited capabilities.
+```{warning}
+The *Reqnroll with Rider* extension cannot work together with the *SpecFlow for Rider* extension, as they both process feature files. As the Reqnroll extension also supports SpecFlow projects, you can remove the SpecFlow extension if you install the Reqnroll extension. Alternatively, you can disable the SpecFlow extension for the time you work with Reqnroll. 
 ```
 
+1. Open Rider and verify that you are using a compatible version. The following versions are verified to work with Reqnroll:
```

Comment:
Seeing as these will change quite regularly I think  it would be better to refer to https://plugins.jetbrains.com/plugin/24012-reqnroll-for-rider/versions for the supported versions.

Reply:
![image](https://github.com/user-attachments/assets/9b636bf2-f319-498d-8352-32bae863b246)